QAT/ubuntu-16.04/media/dev/Dockerfile

```diff
@@ -3,11 +3,14 @@ FROM ubuntu:16.04 AS build
 WORKDIR /home
 
 # COMMON BUILD TOOLS
-RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y -q --no-install-recommends build-essential autoconf make git wget pciutils cpio libtool lsb-release ca-certificates pkg-config bison flex libcurl4-gnutls-dev zlib1g-dev
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y -q --no-install-recommends build-essential autoconf make git wget pciutils cpio libtool lsb-release ca-certificates pkg-config bison flex libcurl4-gnutls-dev zlib1g-dev  && \
```

Comment:
You might want to consider moving the `DEBIAN_FRONTEND=noninteractive` to an env key so that it applies to all of the apt-get commands

Reply:
It is a good suggestion, would integrate it.